app/app/public-cloud/requests/(request)/[id]/request/page.tsx

```diff
@@ -193,7 +193,14 @@ export default publicCloudProductRequest(({ pathParams, queryParams, session, ro
 
   return (
     <div>
-      <PublicCloudBillingInfo product={publicSnap.currentRequest.decisionData} className="mb-2" />
+      <PublicCloudBillingInfo
```

Comment:
It seems you're adding these two fields to align with the type configuration. Instead of adding data, let's omit these two fields from the `Product` type in `PublicCloudBillingInfo`.

Reply:
> I've tested the feature from my local environment and it works as expected.

Did your local environment had a old public cloud data?